diff --git a/src/mp-unitsConfig.cmake b/src/mp-unitsConfig.cmake
index f5bee933d..3bc8f88a0 100644
--- a/src/mp-unitsConfig.cmake
+++ b/src/mp-unitsConfig.cmake
@@ -23,11 +23,11 @@
 include(CMakeFindDependencyMacro)
 
 if(NOT MP_UNITS_API_FREESTANDING AND NOT MP_UNITS_API_STD_FORMAT)
-    find_dependency(fmt)
+    find_dependency(fmt CONFIG)
 endif()
 
 if(MP_UNITS_API_CONTRACTS STREQUAL "GSL-LITE")
-    find_dependency(gsl-lite)
+    find_dependency(gsl-lite CONFIG)
 elseif(MP_UNITS_API_CONTRACTS STREQUAL "MS-GSL")
     find_dependency(Microsoft.GSL)
 endif()
